.gitignore

```diff
@@ -1,9 +1,11 @@
 *.pyc
 venv/
+.arcconfig
```

Comment:
This was quietly intended - this is the configuration location for phabricator that we use internally before going public with any code. I can remove it if you'd prefer.